_docs-2-0/troubleshooting/peformance.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Performance
+category: troubleshooting
+order: 5
+---
+
+Accumulo can be tuned to improve read and write performance.
+
+## Read performance
+
+1. Enable [caching] on tables to reduce reads to disk.
+
+1. Enable [bloom filters][bloom-filters] on tables to limit the number of disk lookups.
+
+1. Decrease the [major compaction ratio][compaction] of a table to decrease the number of
+   files per tablet. Less files reduces the latency of reads.
+
```

Comment:
Adjusting `table.file.compress.blocksize` and `table.file.compress.blocksize.index` can also be helpful.  Lowering `table.file.compress.blocksize` can result in better random seek performance.  However it increases the index size in the file.  If the indexes are too large to fit in cache, this can hinder performance.  Also, as the index size increases the depth of the index tree in each file may increase.  Increasing `table.file.compress.blocksize.index` can reduce the depth of the tree.